src/treq/api.py

```diff
@@ -116,10 +119,19 @@ def request(method, url, **kwargs):
 
 def _client(*args, **kwargs):
     agent = kwargs.get('agent')
+    proxy = kwargs.get('proxy')
+    reactor = default_reactor(kwargs.get('reactor'))
+    pool = default_pool(reactor,
+                        kwargs.get('pool'),
+                        kwargs.get('persistent'))
+    if not agent and proxy is not None:
+        (address, port) = proxy
+        endpoint = HostnameEndpoint(
```

Comment:
adding proxy-authorization header makes it work, but would be much easier to just specify user-password as kwargs to treq, this currently works

```python
def main(reactor, *args):
    d = treq.get('http://httpbin.org/get', proxy=('localhost', 9000),
            headers={"Proxy-Authorization": basic_auth_header('frank', '')})
    d.addCallback(printaa)
    return d
```